starter-code/functions-and-arrays.js

```diff
@@ -23,27 +33,40 @@ var words = [
 var longest = findLongestWord(words);
 console.log(longest);
 
+
 // Calculating a Sum
+var sumTotal = 0;
 function sumArray (array) {
-
+ for(i=0; i<array.length;i+=1){
+   sumTotal += array[i];
+ }
+ return sumTotal;
 }
 
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 var total = sumArray(numbers);
 console.log(total);
 
 // Calculate the Average
+var sumTotal = 0;
 function averageNumbers (array) {
-
+for(i=0; i<array.length;i+=1){
+   sumTotal += array[i];
+ }
+ return sumTotal/=array.length;
```

Comment:
You could call the function you did before to calculate the sum and then divide it for the array.length, you can also use the reduce method for calculating the sum.
Your solution is good though :)